orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton_test.go

```diff
@@ -87,13 +87,16 @@ func TestSingletonRun(t *testing.T) {
 	recvNoCh(t, ch)
 
 	idx0a.AssertExpectations(t)
-	require.Equal(t, reindexSuccessNum, 2)
-	require.Equal(t, reindexDoneNum, 2)
+	require.Equal(t, 2, reindexSuccessNum)
+	require.Equal(t, 2, reindexDoneNum)
 
 	// Test that a network/hardware pair that has been added after Run
 	// will have its states reindexed as well
 
-	reportMoreState(t)
+	// reportAdditionalState reports enough directory records to cause 3 batches per network
+	// (with the +1 gateway status per network). It adds an extra network from 3 -> 4,
+	// so numBatches following this method will be 3 * 4 = 12
```

Comment:
I think it would be good to leave the comment at the function definition, not where it's called.

Reply:
I added some extra comments at the function. However, this comment is specific to what happens in the test.